docs/docs/concepts/password-policy.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: password-policy
+title: Password Policy
+---
+
+Password-based authentication flows are subject to frequent abuse through social
+engineering, password guessing and phishing attacks.  
+Ory Cloud implements measures to provide high security for password-based
+flows.  
+The Ory Cloud password policy is based on recommendations by the
+[`NIST`](https://pages.nist.gov/800-63-3/sp800-63b.html) and
+[`NCSC`](https://www.ncsc.gov.uk/collection/passwords/updating-your-approach)
+institutes among
+[others](https://www.troyhunt.com/passwords-evolved-authentication-guidance-for-the-modern-era/).
+
+- The password must at least be 8 characters long and all characters (unicode,
```

Comment:
This is actually configurable now :) But it is 8 per default

Reply:
I include something about the default and we mention that you can change it in this doc when it is documented how to configure it.